bank2ynab.conf

```diff
@@ -35,7 +35,19 @@ Plugin =
 # Just make a new [section heading] and copy only the lines from above that need a non-default value.
 # Please keep this sorted alphabetically by [country + bank name] for easier maintenance.
 #####################################################################################################
-# DATE FORMATS:
+# FILENAME REGEX DATE FORMATS:
+# YYYY       ([12][0-9]{3})
```

Comment:
Is there a way to turn these samples into actual building blocks that we could use in the `Source Filename Pattern`? 

I.e. could we turn `([12][0-9]{3})` into e.g. `%%m` and so on? If we could, we could shorten:
`(([12][0-9]{3})-(0[1-9]|1[0-2])-(0[1-9]|[12]\d|3[01]))` to just:
`(%%y-%%m-%%d)`. 
This is more elegant and more readable - but we're inventing a new syntax and that may be a bad idea?

Reply:
I think that's an unnecessary amount of work. We'd have to introduce a whole dictionary of terms. Stick with simple regex. Why are we using these crazy long regexes anyway?